src/prefect/deployments.py

```diff
@@ -555,13 +557,24 @@ async def build_from_flow(
             deployment.version = flow.version
         if not deployment.description:
             deployment.description = flow.description
-        if not deployment.storage:
+
+        # proxy for whether infra is docker-based
+        is_docker_based = hasattr(deployment.infrastructure, "image")
+
+        if not deployment.storage and not is_docker_based:
             deployment.path = str(Path(".").absolute())
+        elif not deployment.storage and is_docker_based:
+            # only update if a path is not already set
+            if not deployment.path:
+                deployment.path = "/opt/prefect/flows"
```

Comment:
Trying to understand how to store flows in Docker images (without the need of a local directory such as the one specified here).
From my understanding, you still need to bind your agent "/opt/prefect/flows" directory to the script's "/opt/prefect/flows" dir triggering the flow. So it's not really a Docker image storage. 

Is there any way to specify the `storage` option of `Deployment.build_from_flow()` to store the flow in Docker image ?

Reply:
No need to bind volumes, it's just that the default flow path within the image is set to /opt/prefect/flows, but you can totally change it - this post will likely clarify confusion: https://medium.com/the-prefect-blog/prefect-2-3-0-adds-support-for-flows-defined-in-docker-images-and-github-repositories-79a8797a7371#c74c